packages/web-pkg/src/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const EVENT_TROW_MOUNTED = 'rowMounted'
+export const EVENT_FILE_DROPPED = 'fileDropped'
+export const EVENT_TROW_CONTEXTMENU = 'contextmenuClicked'
+
+export abstract class ImageDimension {
```

Comment:
You only copied this file, right? There is the same file in `web-app-files/src/constants.ts`. Please make sure that this is not duplicated. I.e. please delete the one in `web-app-files` and change all the imports of the file to the `web-pkg` version.